Running broccoli in non-broccoli'ed directory

Since this seems easy to detect, it would be nice to have a better message.
```
/usr/local/lib/node_modules/broccoli-cli/node_modules/resolve/lib/sync.js:32
    throw new Error("Cannot find module '" + x + "' from '" + y + "'");
          ^
Error: Cannot find module 'broccoli' from '/Users/trek'
    at Function.module.exports [as sync] (/usr/local/lib/node_modules/broccoli-cli/node_modules/resolve/lib/sync.js:32:11)
    at Object.<anonymous> (/usr/local/lib/node_modules/broccoli-cli/bin/broccoli:7:28)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:901:3
```

Grunt does a great job of this:
```
grunt-cli: The grunt command line interface. (v0.1.9)

Fatal error: Unable to find local grunt.

If you're seeing this message, either a Gruntfile wasn't found or grunt
hasn't been installed locally to your project. For more information about
installing and configuring grunt, please see the Getting Started guide:

http://gruntjs.com/getting-started
```

--- a/bin/broccoli
+++ b/bin/broccoli
@@ -4,8 +4,19 @@ var resolve = require('resolve')
 
 process.title = 'broccoli'
 
-var broccoliPath = resolve.sync('broccoli', {
-  basedir: process.cwd()
-})
+try {
+  var broccoliPath = resolve.sync('broccoli', {
+    basedir: process.cwd()
+  })
+} catch (e) {
+  console.log('The main Broccoli library was not found.')
+  console.log('')
+  console.log('	If this is an existing project, running `npm install` may solve this.')
+  console.log('	Otherwise, running `npm install broccoli` will install the Broccoli library.')
+  console.log('	Running `npm install --save-dev broccoli` will install the Broccoli library, and save it as a development dependency.')
+  console.log('	Running `npm install --save broccoli` will install the Broccoli library, and save it as a regular dependency.')
+  process.exit(1)
+}
+
 var broccoli = require(broccoliPath)
 broccoli.cli()

